chalice/build_deploy_config.sh

```diff
@@ -22,6 +22,7 @@ if ! aws es describe-elasticsearch-domain --domain-name $dss_es_domain; then
     echo "Please create AWS elasticsearch domain $dss_es_domain or set DSS_ES_DOMAIN to an existing domain and try again"
     exit 1
 fi
+export DSS_VERSION=$(git describe --tags --always)
```

Comment:
I would either move this variable into `environment`, since environment already references it, or move the variable export (`EXPORT_ENV_VARS_TO_LAMBDA`) to the on line 27.

Between these two options. I kinda like the idea of putting it in `environment`.

Reply:
The reason for defining `DSS_VERSION` in this script is to ensure that the version/release tag generated by `git describe --tags --always` is accurate (only and always generated preceding a deploy). If the same can be guaranteed for `source environment`, then I see no problem with moving it into `environment`. @kislyuk thoughts?

What's the second option re: line 27? Not sure I understand

Reply:
Re. DSS_VERSION being in `build_deploy_config.sh`: Ah, I see. You're right.

Re. line 27: This environment variable takes environment variable names to be exported to the lambda environment. The `EXPORT_ENV_VARS_TO_LAMBDA_ARRAY` variable in `environment` serves a similar purpose.

Reply:
Sounds good to me. I agree the logic for determining the version should be here in this file.